flexget/plugins/cli/trakt.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import unicode_literals, division, absolute_import
+
+
+from flexget import options, plugin
+from flexget.event import event
+from flexget.logger import console
+from flexget.manager import Session
+from flexget.plugins.api_trakt import get_access_token, TraktUserAuth, delete_account, PIN_URL
+
+
+def do_cli(manager, options):
+    if options.action == 'auth':
```

Comment:
Each of these if blocks should be own function, then you can call correct function from a dict like

    action_map = {'auth': action_auth, ...}
    action_map[options.action]()